### PR TITLE
Label docker image and remove link that breaks in some contexts.

### PR DESCRIPTION
**Pull request by ghost**
Added a variety of labels to docker image for production of project badges.

Also, links to external files from the `README.md` don't work when the file ends up in docker hub, so removing.

### CODE REVIEW

**Comment by brendo** [comment]
After reading [this issue](https://github.com/docker/hub-feedback/issues/508) and peering into Docker's [automated build image](https://github.com/andyneff/highland_builder/blob/master/builder.py#L33-L55), maybe we should rename the args to match?

Also, the README should probably be updated to show how to build locally with `--build-arg`. It looks like we can also specify default values for these [environment variables](https://docs.docker.com/engine/reference/builder/#/environment-replacement) too.

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,13 @@
+ARG BUILD_DATE
+ARG VCS_REF
+LABEL com.temando.build-date=$BUILD_DATE \
+      com.temando.name="docker-doc-platform" \
+      com.temando.description="Docker image for publishing Temando documentation." \
+      com.temando.vcs-ref=$VCS_REF \
+      com.temando.vcs-url="https://github.com/temando/docker-doc-platform" \
+      com.temando.vendor="Temando Pty Ltd" \
+      com.temando.vcs-type="Git"
```
Comment (brendo) [inline Dockerfile]: Could come from `SOURCE_TYPE`

File: Dockerfile [hunk 2]
```diff
@@ -1,3 +1,13 @@
+ARG BUILD_DATE
+ARG VCS_REF
+LABEL com.temando.build-date=$BUILD_DATE \
+      com.temando.name="docker-doc-platform" \
+      com.temando.description="Docker image for publishing Temando documentation." \
+      com.temando.vcs-ref=$VCS_REF \
+      com.temando.vcs-url="https://github.com/temando/docker-doc-platform" \
```
Comment (brendo) [inline Dockerfile]: Could come from `SOURCE_URL`

File: Dockerfile [hunk 3]
```diff
@@ -1,3 +1,13 @@
+ARG BUILD_DATE
+ARG VCS_REF
+LABEL com.temando.build-date=$BUILD_DATE \
+      com.temando.name="docker-doc-platform" \
+      com.temando.description="Docker image for publishing Temando documentation." \
+      com.temando.vcs-ref=$VCS_REF \
```
Comment (brendo) [inline Dockerfile]: Rename to `SOURCE_COMMIT`?

File: Dockerfile [hunk 4]
```diff
@@ -1,3 +1,13 @@
+ARG BUILD_DATE
+ARG VCS_REF
+LABEL com.temando.build-date=$BUILD_DATE \
```
Comment (brendo) [inline Dockerfile]: Drop `build-date`?

File: Dockerfile (hunk 1 above)
Comment (ghost) [reply Dockerfile]: Better. Fixed.

File: Dockerfile (hunk 2 above)
Comment (ghost) [reply Dockerfile]: Better. Fixed.

File: Dockerfile (hunk 3 above)
Comment (ghost) [reply Dockerfile]: Better. Fixed.

File: Dockerfile (hunk 4 above)
Comment (ghost) [reply Dockerfile]: Happy to drop it. Done.

**Pull request merged**